cluster/v1/types.go

```diff
@@ -67,6 +67,10 @@ type ManagedClusterSpec struct {
 	// +optional
 	// +kubebuilder:default=60
 	LeaseDurationSeconds int32 `json:"leaseDurationSeconds,omitempty"`
+
+	// Taints is a property of managed cluster that allow the cluster to be repelled when scheduling.
```

Comment:
Describe which actors you are expecting to have rights to remove a taint.  Does an agent running in the spoke cluster have the power to remove a taint?  Any taint?

Nodes are forbidden to modify taints to avoid steering workloads based on compromised node credentials (see https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/noderestriction/admission.go#L428-L432)

I'm open to reconsidering this while starting from scratch by asserting that Taints are only for anti-selection and therefore could be set by any agent, but we should explicitly state that here.

Reply:
Doc updated for the clarification.